src/models/Todo.tsx

```diff
@@ -0,0 +1,8 @@
+export interface Todo {
+    id:string,
+    Name: string,
+    Text: string,
+    DueDate: Date,
+    Priority:string,
```

Comment:
```suggestion
    Priority:"Low"|"Normal"|"High",
```
https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types

Reply:
camelCase is more natural in js